schemas/Clinical/index.js

```diff
@@ -141,6 +143,25 @@ const typeDefs = gql`
     donorId: String!
   }
 
+  type ClinicalSubmissionError implements ClinicalEntityError @cost(complexity: 5) {
```

Comment:
rename to `ClinicalSubmissionDataError`?

Reply:
addressed :)